tests/test_suite.py

```diff
@@ -109,6 +111,16 @@
 # debug otherwise it cuts silently everything
 log.setLevel(logging.DEBUG)
 
+try:
+    bench_id = xml_operations.infer_benchmark_id_from_component_ref_id(
```

Comment:
why not use the component ref directly? oscap supports it:

```
--xccdf-id ID
                     Takes component ref with given ID from checklists. This allows to select a particular XCCDF component even in cases where there are 2 XCCDFs in  one  datastream.  If
                     none is given, the first component from the checklists element is used.
```

Reply:
It seems to me that you can refactor the code to avoid the whole infer id code. Instead of looking for //cdf:Benchmark you can look for ds:component[@id=...]/cdf:Benchmark. Or am I missing something? Would make the code way shorter.

Reply:
I still see need for a ref-id -> benchmark function, although it has potential be a one-liner (if one knows how to write it and the module supports the required part of the XPath language).